### PR TITLE
:building_construction: Externalize export in its own method

### PR DESCRIPTION
**Pull request by Errorname**

### CODE REVIEW

File: src/Mustaches.ts [hunk 1]
```diff
@@ -16,8 +23,7 @@ class Mustaches {
     destination,
     name,
     data,
-    formatters = {},
-    export: exportType
+    formatters = {}
   }: InterpolationOptions): Promise<ID> {
     // If no destination given, use same folder as source
     destination = destination || (await this.getParent(source))
```
Comment (antoinecarat) [inline src/Mustaches.ts]: Is this still needed?

Comment (Errorname) [reply src/Mustaches.ts]: Yes, you may want to put the export in another folder

Comment (antoinecarat) [reply src/Mustaches.ts]: Sure, but you don't need the destination in `interpolate` anymore

Comment (Errorname) [reply src/Mustaches.ts]: We still need it for the interpolated Google Doc file: Template file => Interpolated file => Exported file. The interpolated and exported files need to be put somewhere

Comment (antoinecarat) [reply src/Mustaches.ts]: Oh yes sorry, I forgot this "intermediate" file, my bad

**Pull request merged**